advanced_functionality/working_with_tfrecords/tfrecord-transformer-container/entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/env bash
+set -e
+
+# Get the listen port from the SM env variable, otherwise default to 8080
+LISTEN_PORT=${SAGEMAKER_BIND_TO_PORT:-8080}
+
+# Set the number of gunicorn worker processes
+GUNICORN_WORKER_COUNT=16
```

Comment:
the right number of workers will vary based on instance type. 

using `GUNICORN_WORKER_COUNT=$(nproc)` will set the number of workers to be equal to the number of vcpus, which is usually the right answer.

Reply:
fixed